homeassistant/components/automation/services.yaml

```diff
@@ -12,6 +12,9 @@ turn_off:
     entity_id:
       description: Name of the automation to turn off.
       example: "automation.notify_home"
+    stop_actions:
```

Comment:
Is `stop_actions` the right term? Maybe `cancel_current` fits better, since `current` is also the attribute that shows the number of currently active invocations ?

Reply:
Well, the actions are in the `action` section of the automation, and all the documentation talks about "actions." I don't know. Half a dozen of one, six of the others.